app/helpers/service_helper.rb

```diff
@@ -34,10 +38,17 @@ def switch_service_link
     link_to(t('service_banner.service_name.uk'), current_path)
   end
 
-  def service_switch_banner(optional_classes: 'govuk-!-margin-bottom-7')
+  def service_switch_banner
     # rubocop:disable Rails/HelperInstanceVariable
     if @enable_service_switch_banner_in_action
-      tag.div(class: "tariff-breadcrumbs js-tariff-breadcrumbs clt govuk-!-font-size-15 #{optional_classes}") do
+      css_classes = %w[tariff-breadcrumbs
+                       js-tariff-breadcrumbs
+                       clt
+                       govuk-!-font-size-15
```

Comment:
Would it make sense to `@extend` these in the `tariff-breadcrumbs` css class ? Or better yet `@include govuk-typography-responsive($size: 15);`